GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java

```diff
@@ -437,25 +437,26 @@ private static eUnits get_eUnits(String unit) {
         }
     }
 
-    private String translate(String chinese) {
-        switch (chinese) {
-            case "公里":
-            case "킬로미터":
-                return "km";
-            case "公尺":
-            case "미터법":
-                return "m";
-            case "分":
-            case "分鐘":
-            case "분":
-                return "m";
-            case "小時":
-            case "時":
-            case "시간":
-                return "h";
-            default:
-                return null;
-        }
+    // Translates the units (distance and time) from local language and charset in common values
+    private String translate(String local_language_string) {
+        if(local_language_string == getString(R.string.km))
```

Comment:
Nice job.
I want to change the string judgement by R.string, but switch/case cannot support dynamic values.
I forgot if /else can do this.......

Reply:
Thanks
Yes had the same problem, but then found that solution